README.md

```diff
@@ -187,6 +191,7 @@ The `androidApkUpload` build step lets you upload Android App Bundle (AAB) or AP
 | expansionFilesPattern              | string  | `'**/*.obb'`           | (none)                                                   | Comma-separated glob patterns or filenames pointing to expansion files to associate with the uploaded APK files        |
 | usePreviousExpansion<br>FilesIfMissing | boolean | `false`            | `true`                                                   | Whether to re-use the existing expansion files that have already been uploaded to Google Play for this app, if any expansion files are missing |
 | recentChangeList                   | list    | (see below)            | (empty)                                                  | List of recent change texts to associate with the upload app files                                                     |
+| inAppUpdatePriority                | string  | `'1'`                  | (none)                                                   | Priority used for In-App update feature                                                                                |
```

Comment:
Yeah at first I wanted it to be a Integer but I want also to be possible to use environment variable since this might come from for instance input parameter